ontology/yaml/resources/HVAC/entity_types/MAU.yaml

```diff
@@ -954,6 +954,18 @@ MAU_DFSS2X_DFVSC2X_CHWZC_HWZC_VOADM:
   - HWZC
   - VOADM
 
+MAU_DFVSC_CHWDDC_OAMC_OA:
+  guid: "1a52f54a-6e0f-4565-83cf-90be0d8f4d3c"
+  description: "Single zone DOAS with damper and discharge fan speed control."
+  is_canonical: true
+  implements:
+  - DOAS
+  - DFVSC
+  - DSPM
```

Comment:
this is not listed in the canonical name

Reply:
I have updated the list for the MAU type